### DISCUSSION TITLE
Exporting site from wayback machine to archive box

### DISCUSSION THREAD

**Discussion by mscottco** [ArchiveBox/ArchiveBox#972] [Q&A]
Hi All,

I was wondering if there is a way I could save a local copy of all pages under a given domain from the wayback machine to an archive box. Ideally I'd need to get the complete history and ideally maintain all the timestamps within the archive box created... I've considered using tools like waybackpack although as I understand it, I'd loose all the timestamps etc.

Hope this is an okay place to ask,

Cheers!

**Comment by pirate** [comment]
Nope, ArchiveBox doesn't have any feature to do this, and generally isn't designed for archiving entire domains. Check out the alternatives in our wiki/REAMDE. 

Or follow this issue for progress: https://github.com/ArchiveBox/ArchiveBox/issues/160
